### DISCUSSION TITLE
Page Rules on Groups: Limiting by "Tags" won't work on assets

### DISCUSSION THREAD

**Discussion by KingRial** [requarks/wiki#3590] [Error / Bug Report]
**Describe the bug**
When configuring "Page Rules" the administration allow to limit access by "Tags" (which is, by far, the most intuitive and fast way to configure access rights).

However these limits won't work correctly on assets added to pages.

**To Reproduce**
1. Access Administration
2. Create a User Group
3. Add a simple page rule based on Tag matches
4. Create a User and assign it to the User Group
5. Create a Page and use the correct tag
6. Add an asset to the page
7. Access the page with the created user. Page should be accessible while the asset shouldn't

**Expected behavior**
Assets should have tags too or being able to inherit them from pages on which are inserted

**Screenshots**
![image](https://user-images.githubusercontent.com/886518/88168627-fc294300-cc1a-11ea-9670-d1615619f2e4.png)
As shown in the previou screenshot, to limit access to the assets I had to:

- build a specific tree structure on which upload the assets
- add more rules, based on paths, to allow the correct access rights to the assets

**Host Info:**
 - OS: Linux, Debian 10, Docker
 - Wiki.js version: 2.4.107
 - Database engine: postgres 9.7

**Additional context**

**Comment by leomarques3** [comment]
I'm with the same problem, when I tag a group, the same can't view or download assets. Did you find a solution?

**Comment by KingRial** [comment]
@leomarques3 I figured how and where to add the feature; then "life happened" and I lost the time to code solutions on this project ^^

I just moved to limiting access by "path" instead of using tags (even If I really liked tha tag usage to limit access)
